common/djangoapps/student/helpers.py

```diff
@@ -425,6 +427,8 @@ def authenticate_new_user(request, username, password):
     logged in until they close the browser. They can't log in again until they click
     the activation link from the email.
     """
+    if PASSWORD_UNICODE_NORMALIZE.is_enabled():
```

Comment:
It is mentioned elsewhere about other ways to phrase this for clarity. Want to add a reminder to stay consistent after making any changes.